frontend/src/components/Patchnotes.jsx

```diff
@@ -1,14 +1,28 @@
 import React, { useEffect, useState } from 'react';
 import { useParams } from 'react-router-dom';
 
-const Patchnotes = () => {
-    const { year, month } = useParams();
-    const [patchDetails, setPatchDetails] = useState({ tank: [], damage: [], support: [], mapUpdates: [], bugFixes: [] });
+const Patchnotes = ({ game }) => {
+    const { year, month, version } = useParams();
+    const [patchDetails, setPatchDetails] = useState({
+        tank: [],
+        damage: [],
+        support: [],
+        agentUpdates: [],
+        mapUpdates: [],
+        bugFixes: []
```

Comment:
this seems specific to Overwatch, is that right?

I think there's a good opportunity to either:
- create an abstraction over the data across different games (e.g. PatchSection { title, content })
- or just embrace the difference and create different components for each Game